### PR TITLE
[ADD] sale_product_packaging_container_deposit

### PR DESCRIPTION
**Pull request by santostelmo**
[ADD] sale_product_packaging_container_deposit

Depends on:
- https://github.com/OCA/product-attribute/pull/1418

### CODE REVIEW

File: sale_product_packaging_container_deposit/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2023 Camptocamp (<https://www.camptocamp.com>).
```
Comment (simahawk) [inline sale_product_packaging_container_deposit/__manifest__.py:1]:
```suggestion
# Copyright 2023 Camptocamp SA (<https://www.camptocamp.com>).
```
everywhere for (c)

**Review by simahawk — CHANGES_REQUESTED** [review]
add test-requirements pls

**Comment by santostelmo** [comment]
> add test-requirements pls

Test requirements added.

**Review by cyrilmanuel — APPROVED** [review]

**Comment by simahawk** [comment]
> > add test-requirements pls
> 
> Test requirements added.

you can remove it now ;)

**Comment by santostelmo** [comment]
> > > add test-requirements pls
> > 
> > 
> > Test requirements added.
> 
> you can remove it now ;)

Removed test requirements

**Pull request merged**